### PR TITLE
ci: Remove azure-based workers pipelines

### PR DESCRIPTION
**Pull request by ldoktor**
the github free-runners seem to perform well and stable, let's disable the azure runners.

### CODE REVIEW

**Review by stevenhorsman — APPROVED** [review]
Makes sense to me. Thanks @ldoktor!

**Comment by mythi** [comment]
is this dropping any of the targets that keeps failing for me in #429?

**Comment by stevenhorsman** [comment]
Once we have two approvals I guess we should remove the az jobs from the required list and add the free runners?

**Comment by stevenhorsman** [comment]
It looks like the `s390x-large` runner is offline and @BbolroC is on vacation for a few days, so we'll need to ignore that job too

**Comment by ldoktor** [comment]
> Once we have two approvals I guess we should remove the az jobs from the required list and add the free runners?

Yep, actually let me do that now since both jobs are currently being executed and the az ones are now actually less reliable than the GH ones.

**Pull request merged**
